setup.py

```diff
@@ -101,7 +101,6 @@
             "bowtie1 = multiqc.modules.bowtie1:MultiqcModule",
             "bowtie2 = multiqc.modules.bowtie2:MultiqcModule",
             "busco = multiqc.modules.busco:MultiqcModule",
-            "bustools = multiqc.modules.bustools:MultiqcModule",
```

Comment:
Was this an accident? (looks like it)

Reply:
These were accidental, I'm gonna re-add them back.